task/buildpacks/0.2/buildpacks.yaml

```diff
@@ -106,6 +106,9 @@ spec:
           mountPath: /cache
         - name: $(params.PLATFORM_DIR)
           mountPath: /platform
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
```

Comment:
@vinamra28 

Are these magic values for a particular platform or am I mistaken in thinking that these should be the values provided via params (`$(params.USER_ID)` and `$(params.GROUP_ID)`)?

Reply:
Yeah these values should have been provided via `params` but in the current sceanrio Tekton `params` doesn't support `integer` data type and `runAsUser` requires an integer type value hence I hard coded this for now but yeah in future we need to fix this.